cmd/lit-af/shell.go

```diff
@@ -223,14 +225,42 @@ func (lc *litAfClient) Ls(textArgs []string) error {
 		return err
 	}
 	if len(cReply.Channels) > 0 {
-		fmt.Fprintf(color.Output, "\t%s\n", lnutil.Header("Channels:"))
+		fmt.Fprintf(color.Output, "\t%s\n", lnutil.Header("Closed Channels:"))
 	}
 
+	sort.Slice(cReply.Channels, func(i, j int) bool {
+		return cReply.Channels[i].Height < cReply.Channels[j].Height
+	})
+
+	var closedChannels []litrpc.ChannelInfo
+	var openChannels []litrpc.ChannelInfo
 	for _, c := range cReply.Channels {
 		if c.Closed {
-			fmt.Fprintf(color.Output, lnutil.Red("Closed  "))
+			closedChannels = append(closedChannels, c)
+		} else {
+			openChannels = append(openChannels, c)
+		}
+	}
+
+	for _, c := range closedChannels {
```

Comment:
Maybe swap the channel sets and put the closed ones at the bottom? Makes sense to put the active ones on top as it sort of naturally feels that whatever is on top is more important.

Reply:
Will do

Reply:
Okay, just remembered something, why I put it on top was because people would have to scroll up and see the open channels otherwise.

Reply:
Okay, that stems from the fact that `ls` outputs so much at once. Created an issue #230 to try and fix that. I think we can resolve this remark that way too, and for this PR the chosen sequence makes sense.